### DISCUSSION TITLE
Problems with step 2 of CloudFormation

### DISCUSSION THREAD

**Discussion by Roberto-cpu-spec** [mozilla/hubs#3725] [Hubs Cloud Questions] [answered]
Hi everyone, i need your help, i try to implement hubs cloud personal, i follow all the steps of the video guides, but in the step 2 of CloudFormation, when i click on the button (next) , show a page with the message "oops something will wrong - could not load the page. refresh the page and try again" I try many times and the result is the same
![HC](https://user-images.githubusercontent.com/74120631/104984658-c7812300-59d4-11eb-98d3-1711459efd28.png)

**Comment by robinkwilson** [comment, marked as answer]
Interesting! I've never seen this before on AWS. 

Can you take a screenshot of the page before you get this page (with the "Next" button)?

**Reply by Roberto-cpu-spec** [reply]
Thank you very much for answer me (I think you are the best), I honestly thought that nobody would answer me, in fact last night I was about to send you an email; my problem has already been solved, I think the possible problems were:
 ▪︎ the payment of my domains on route 53 had not been applied
 ▪︎ have the page with translation into Spanish
 ▪︎24 hours had to pass

 Finally I already have personal hubscloud (following your steps)
 Now I have other doubts, I received an email from aws saying that it "exceeded 85% of the usage limit for one or more AWS Free Tier-eligible services for the month of January". I don't know if it is something related to hubs.
![AWS](https://user-images.githubusercontent.com/74120631/105311616-166cbb00-5b83-11eb-9b41-1d1dce9224db.jpg)



 But my main doubts are regarding the operation of hubs.
 I hope write to you regarding many doubts that i have about HubsCloud.  
I am a teacher and my goal with hubs is educational.
 An apology for my writing, I am not fluent in the English language.

**Reply by robinkwilson** [reply]
For the usage limit, I'd shrink the instance size to t3.small if you're using it for small gatherings and not hosting large events. AWS gives you an EC2 instance for free for a time. You'll be billed after it exceeds the limit.

**Reply by Roberto-cpu-spec** [reply]
Thanks for the support, I do not understand very well what an instance is and what exceeding the limit refers to, but I will continue investigating.
For now I entered my AWS console and I see that I have the .t3small instance, I hope to continue learning.
Thanks a lot.

![CONSOLA](https://user-images.githubusercontent.com/74120631/106369545-19e90a80-6318-11eb-9d37-abb46f35a544.png)

**Comment by paula-soares** [comment]
Hi, I'm Paula, I'm in Brazil. I'm having the same problem, I've been following the same tutorial on youtube, I registered the domains for two days, but they are ok as shown here. I put some prints.

**Comment by paula-soares** [comment]
![5](https://user-images.githubusercontent.com/58034726/115790348-f4aaec00-a39c-11eb-9752-60f0d158d7c7.png)
![4](https://user-images.githubusercontent.com/58034726/115790350-f5438280-a39c-11eb-921b-856b5118776a.png)
![3](https://user-images.githubusercontent.com/58034726/115790352-f5438280-a39c-11eb-9d63-126709496341.png)
![2](https://user-images.githubusercontent.com/58034726/115790354-f5dc1900-a39c-11eb-930e-7a3a791ea2d4.png)
![1](https://user-images.githubusercontent.com/58034726/115790355-f5dc1900-a39c-11eb-81c5-de96c3331f57.png)
![8](https://user-images.githubusercontent.com/58034726/115790356-f5dc1900-a39c-11eb-9136-0c8867c091f0.png)
![7](https://user-images.githubusercontent.com/58034726/115790357-f674af80-a39c-11eb-89eb-43e301d91035.png)
![6](https://user-images.githubusercontent.com/58034726/115790359-f674af80-a39c-11eb-84c8-769e4af379b9.png)

**Reply by Roberto-cpu-spec** [reply]
Hi Paula, I'm not sure of the solution but apparently it was because I had activated the translator of the page in the browser, I see in your images that you have it too, I deactivated the translation and it worked for me. turn off the translator or go back to the English language.

**Reply by paula-soares** [reply]
certo vou tentar isso! Obgda!

**Comment by paula-soares** [comment]
it worked, thank you!

**Reply by Roberto-cpu-spec** [reply]
Excellent, you're welcome.
